src/app/components/card/card.component.html

```diff
@@ -0,0 +1,53 @@
+<div class="card-container">
+    <div class="tds-headline-02 tds-u-pb1">Card</div>
+    <tds-card header="Card" subheader="Simple Card">
+      <div slot="body">This is a simple Card example</div>
+    </tds-card>
+    <div class="split-cards">
+      <tds-card header="Card" subheader="Card with image">
+        <img
+          class="tds-u-w-100"
+          slot="body-image"
+          alt="Scania graphical lettering sign"
+          src={{cardImagePath}}
+        />
+        <div slot="bottom">
+          <p class="tds-text-grey-958">
+            The bottom section can have a
+            <tds-link>
+              <a href="/"> link.</a>
+            </tds-link>
+            This Card has its image-placement below-header.
+          </p>
+        </div>
+      </tds-card>
```

Comment:
Can you please also add one example where we use prop for image instead of slot? 
Just to test that functionality too.